code/modules/mob/living/carbon/human/species/station/station_special_vr.dm

```diff
@@ -337,6 +337,9 @@
 	icobase_tail = 1
 
 	inherent_verbs = list(
```

Comment:
Please comment block this with //YW Changes and //END YW Changes or something similar since this isn't a _yw file. Thanks.

Reply:
you want me to show what was changed ?, sorry a bit new to GitHub in general